eng/build.yml

```diff
@@ -28,6 +28,12 @@ jobs:
       Release:
         BuildConfiguration: Release
   steps:
+  - task: DotNetCoreCLI@2
```

Comment:
Should not longer be needed,  please merge with master

Reply:
Done